### PR TITLE
Create image label checker

### LINKED ISSUE #45: Expand semantic label checker to images
Images should also have semantic labels to ensure accessibility.

There is a property in the `Image` widget called `semanticLabel`, I would believe checking it would be similar to the button one.

### PR DESCRIPTION
**Pull request by phcs971**
Using the same logic for input label checker, created a new image label checker that checks if SemanticFlag isImage is on.

Also works on SvgPicture

Fix #45

### CODE REVIEW

File: lib/src/accessibility_tools.dart [hunk 1]
```diff
@@ -49,6 +50,7 @@ class AccessibilityTools extends StatefulWidget {
     this.checkSemanticLabels = true,
     this.checkMissingInputLabels = true,
     this.checkFontOverflows = false,
+    this.checkImageLabels = false,
```
Comment (aednlaxer) [inline lib/src/accessibility_tools.dart:53]: I think we should turn it on by default, this is quite an important check

**Review by aednlaxer — COMMENTED** [review]
Looks really good, just needs tests and coverage fixed

**Comment by phcs971** [comment]
> Looks really good, just needs tests and coverage fixed

Fixed!

**Comment by phcs971** [comment]
@aednlaxer can we run the CI again, it was an error on publishing the coverage

**Comment by aednlaxer** [comment]
> @aednlaxer can we run the CI again, it was an error on publishing the coverage

Sure thing. However, a rate limit has been reached, I'll take a look at it

**Comment by phcs971** [comment]
> > @aednlaxer can we run the CI again, it was an error on publishing the coverage
> 
> Sure thing. However, a rate limit has been reached, I'll take a look at it

I see, might have to wait a little then, time to availability says 22 min

**Pull request merged**